### ISSUE TITLE
NTR: CT or MRI: spinal intradural lipoma (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8922]
New term request CT or MRI: spinal intradural lipoma (segal_100730170157):   UMLS: 0347445

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000415
- label: Intradural spinal lipoma
- parents: [Spinal cord tumor (HP:0010302)](https://hpo.jax.org/app/browse/term/HP:0010302)
- pmid: PMID:21887183
- def: Intradural spinal lipoma (ISL) constitutes approximately 1% of all spinal tumors and is usually associated with lumbo-sacral spinal dysraphism. ISLs unassociated with spinal dysraphism are rare with only a few cases being reported in literature. They are seen in children and young adults and are situated on the posterior aspect of the spinal cord in the cervico-thoracic or thoracic regions. They represent intradural, subpial, juxtamedullary lesions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8922
- synonyms:
